docs/documentation/user.md

```diff
@@ -83,10 +86,40 @@ can always ask in
 [#documentation](https://chat.zulip.org/#narrow/stream/19-documentation)
 on the [Zulip community server](https://zulip.com/development-community/).
 
+### Updating an existing article
+
+If the new feature you're documenting is a refinement on,
+or related to, a feature that already has a dedicated help center
+article, the information will be more useful and discoverable for
+users as an addition to the existing article.
+
+Here are some things to keep in mind when expanding and updating
+existing help center articles:
+
+- Maintain the format and style of the current article.
+
+- Review the [Markdown features](#markdown-features) available and
+  pay attention to the ones already utilized in the article.
+
+- Think about where inline links to other help center documentation
+  would be appropriate in the description of the feature. For example,
+  your new feature might relate to general Zulip features like
+  [keyboard shortcuts](https://zulip.com/help/keyboard-shortcuts)
+  or [streams and topics](https://zulip.com/help/streams-and-topics).
+
+- Make sure there is a **Related articles** section at the end
+  of the article that again links to any help center documentation
+  mentioned in the text or related to the feature.
+
+Creating robust and informative help center articles with good links
+will allow users to navigate the help center much more effectively
+and efficiently.
```

Comment:
I would reword this to, "navigate the help center more effectively" for brevity.